_dev/src/App.vue

```diff
@@ -72,6 +72,18 @@
       MenuItem,
       RoundingBanner,
     },
+    data() {
+      return {
+        paypalStatusUpdater: null,
+      };
+    },
+    methods: {
+      updater() {
+        this.paypalStatusUpdater = setInterval(() => {
+          this.$store.dispatch('updatePaypalStatus');
+        }, 5000);
```

Comment:
This value seems low, I guess we can double it to 10s.

Reply:
Changed to 10 seconds.